### PR TITLE
Add django-controlcenter

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge-admin : Since it has passed all checks, could you please review it and merge.

File: recipes/example/django-controlcenter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    
+test:
+  imports:
+    - controlcenter
+    - tests
```
Comment (isuruf) [inline recipes/example/django-controlcenter/meta.yaml]: ?

Comment (isuruf) [reply recipes/example/django-controlcenter/meta.yaml]: Removing this from here is not enough. You need to remove them from the conda package and also raise an issue upstream

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge

File: recipes/example/django-controlcenter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    
+test:
+  imports:
+    - controlcenter
```
Comment (CJ-Wright) [inline recipes/example/django-controlcenter/meta.yaml]: More tests please

**Comment by CJ-Wright** [comment]
Seems reasonable.

File: recipes/example/django-controlcenter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/byashimov/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipes/example/django-controlcenter/meta.yaml]: Would it be possible to use pip? https://conda-forge.org/docs/meta.html#use-pip

Comment (sannykr) [reply recipes/example/django-controlcenter/meta.yaml]: sure. Thanks for reviewing it.

**Comment by jakirkham** [comment]
So this got added to the `example` directory. 😕 Also means that the recipe was not actually tested on CI. @sannykr, could you please submit a PR to move this out of the `example` directory?

**Comment by isuruf** [comment]
You'll also need to address my comment at https://github.com/conda-forge/staged-recipes/pull/5070#discussion_r165817133

**Comment by CJ-Wright** [comment]
Sorry :disappointed:

**Comment by sannykr** [comment]
@jakirkham : I am trying to make a different pull request, but it has already created a feedstock. Will it cause any issue later?

**Pull request merged**